subsys/usb/device_next/class/usbd_cdc_ncm.c

```diff
@@ -0,0 +1,1372 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
```

Comment:
Not sure we need to keep "Copyright (c) 2017 Intel Corporation", looks like here is finally nothing more reused from the legacy implementations.